cookbook/ifan02.rst

```diff
@@ -0,0 +1,181 @@
+Ifan02
+======
+
+.. seo::
+    :description: Instructions for using Sonoff ifan02 in ESPHome.
+    :keywords: Fan, Sonoff, ifan02
```

Comment:
This needs an `:image:` too.

Reply:
Hmm where to find a good free non copyrighted image ? Mine are installed so i can't take a picture.

Reply:
i just used the fan.svg